README.md

```diff
@@ -185,8 +185,8 @@ Then wire it all up in your `features/support/env.rb` file:
 require 'aruba'
 require 'aruba/in_process'
 
-Aruba::InProcess.main_class = MyMain
-Aruba.process = Aruba::InProcess
+Aruba.process = Aruba::Processes::InProcess
```

Comment:
There are quite a few classing around starting processes. That's why I namespaced them.